website/source/docs/connect/mesh_gateway.html.md

```diff
@@ -24,6 +24,16 @@ Each mesh gateway needs three things:
 2. General network connectivity to all services within its local Consul datacenter.
 3. General network connectivity to all mesh gateways within remote Consul datacenters.
 
+Mesh gateways also require that your Consul datacenters are configured correctly:
+
+- You'll need to use Consul version 1.6.0-beta1.
+- Consul [Connect](/docs/agent/options.html#connect) must be enabled in both datacenters.
+- Each of your [datacenters](/docs/agent/options.html#datacenter) must have a unique name. 
+- Your datacenters must be [WAN joined](https://learn.hashicorp.com/consul/security-networking/datacenters).
+- The [primary datacenter](/docs/agent/options.html#primary_datacenter) must be set to the same value in both datacenters. This specifies which datacenter is the authority for Connect certificates and is required for all datacenters to trust requests from each other.
```

Comment:
```suggestion
- The [primary datacenter](/docs/agent/options.html#primary_datacenter) must be set to the same value in both datacenters. This specifies which datacenter is the authority for Connect certificates and is required for all datacenters to trust Connect requests from each other.
```

Maybe it's worth being clear that this is only needed for Connect requests. RPC would still work for example.

Reply:
Would it be accurate to say 

"and is required for services in all datacenters to establish mutual TLS with each other?" 

"Connect" is a bit overloaded and so I think it might be better to be more specific :)

Reply:
Yeah I think that is good!